book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
```

Comment:
- [ ] A method for adding rentals should be added to the Book class, please review [this](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars).

Reply:
Well resolved 👍🏽